src/apps/create-patron-user-info/CreatePatron.dev.tsx

```diff
@@ -14,6 +14,16 @@ export default {
     ...pincodeArgs,
     ...globalTextArgs,
     ...globalConfigArgs,
+    authUrl: {
```

Comment:
Maybe we should consider calling this `loginUrl`?

Reply:
It has been named like this since the dawn of time. Would require a refactor in the cms as well. Lets keep that for a rainy day ;) But good point!

Reply:
Fair enough 😄